core/src/main/java/com/adobe/aem/commons/assetshare/content/renditions/impl/dispatchers/StaticRenditionDispatcherImpl.java

```diff
@@ -129,12 +143,52 @@ public void dispatch(SlingHttpServletRequest request, SlingHttpServletResponse r
         }
     }
 
+    private void dispatchFromAemCloud(Rendition rendition, AssetRenditionParameters parameters, SlingHttpServletResponse response) throws IOException, ServletException {
+        if (rendition != null && rendition.getBinary() != null && rendition.getBinary() instanceof BinaryDownload) {
+            log.debug("Redirecting rendition [ {} ] for resolved rendition name [ {} ] from the Adobe Cloud", rendition.getPath(), parameters.getRenditionName());
+
+            final BinaryDownload binaryDownload = (BinaryDownload) rendition.getBinary();
+
+            BinaryDownloadOptions downloadOptions;
+
+            if (parameters.isDownload()) {
+                // Mark disposition type as Attachment, to invoke download in browser
+                downloadOptions = BinaryDownloadOptions.builder()
+                        .withMediaType(rendition.getMimeType())
+                        .withFileName(parameters.getFileName())
+                        .withDispositionTypeAttachment()
```

Comment:
@bobvanmanen what can I do prior to the Sling Include to ensure internally in Sling `. withDispositionTypeAttachment` or `. withDispositionTypeInline` is appropriately applied (Keep in mind, ASC allows this to be specified at a high-level abstraction)